get-started-with-tidb-lightning.md

```diff
@@ -1,54 +1,61 @@
 ---
-title: TiDB Lightning 教程
+title: TiDB Lightning 快速上手
 aliases: ['/docs-cn/dev/get-started-with-tidb-lightning/','/docs-cn/dev/how-to/get-started/tidb-lightning/']
 ---
 
 # TiDB Lightning 快速上手
 
-本教程假设使用的是若干新的、纯净版 CentOS 7 实例，你可以（使用 VMware、VirtualBox 及其他工具）在本地虚拟化或在供应商提供的平台上部署一台小型的云虚拟主机。因为 TiDB Lightning 对计算机资源消耗较高，建议分配 16 GB 以上的内存以及 32 核以上的 CPU 以获取最佳性能。
+本文档介绍如何快速上手 TiDB Lightning，将 MySQL 数据导入到 TiDB 集群中。
 
 > **警告：**
 >
 > 本教程中的部署方法只适用于测试及功能体验，并不适用于生产或开发环境。
 
-## 准备全量备份数据
+## 第 1 步：准备全量备份数据
 
-我们使用 [`dumpling`](/dumpling-overview.md) 从 MySQL 导出数据，如下：
+你可以使用 [`dumpling`](/dumpling-overview.md) 从 MySQL 导出数据。
 
-{{< copyable "shell-regular" >}}
+1. 安装 Dumpling：
 
-```sh
-tiup dumpling -h 127.0.0.1 -P 3306 -u root -t 16 -F 256MB -B test -f 'test.t[12]' -o /data/my_database/
-```
+    ```shell
+    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh && \
```

Comment:
This command is to install TiUP, right ?  
Maybe you can make it as a separate step to verify whether TiUP is installed.  If not , use `curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh` to install the TiUP

Reply:
Fixed in [66194be](https://github.com/pingcap/docs-cn/pull/12059/commits/66194bef4b858dbf019518aad9d8dc51756742f6)